modules/vulnerability-management20-delete-reports.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// * operating/manage-vulnerabilities.adoc
+:_content-type: PROCEDURE
+[id="vulnerability-management20-delete-reports_{context}"]
+= Deleting vulnerability reports
+
+[role="_abstract"]
+Deleting a report deletes the report and any reports that were previously run.
```

Comment:
Deleting a report configuration deletes the config and any report jobs that were previous run using this configuration.